### PR TITLE
DEP remove appveyor

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender.

**Comment by beckermr** [comment]
So our new VC package is not working here: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=137685&view=logs&j=f97ca392-d626-52d0-4b2e-4b27aa319a77&t=3002d0ca-d910-5264-f0e4-809a5057fac4

cc @jjhelmus @isuruf

**Comment by isuruf** [comment]
I remember that the latest vs2017_win-64 from defaults not working. That's why we copied the last working version. https://anaconda.org/conda-forge/vs2017_win-64/files
We should revert to the AnacondaRecipes recipe for build number 4

**Comment by isuruf** [comment]
Need to add https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-isl/isl-0.14.1-no-undefined.patch

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/no_undefined.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -164,7 +164,7 @@
+ 	isl_vertices_private.h \
+ 	isl_vertices.c
+ libisl_la_LIBADD = @MP_LIBS@
+-libisl_la_LDFLAGS = -version-info @versioninfo@ \
++libisl_la_LDFLAGS = -version-info @versioninfo@ -no-undefined \
```
Comment (beckermr) [inline recipe/no_undefined.patch:8]: Does this one need to have `--no-undefined` and not `-no-undefined`? That was the latest build error.

**Comment by beckermr** [comment]
ugh

```
cd . && /bin/sh /d/bld/isl_1586871396987/work/missing automake-1.15 --foreign Makefile
configure.ac:4: error: version mismatch.  This is Automake 1.15,
configure.ac:4: but the definition used by this AM_INIT_AUTOMAKE
configure.ac:4: comes from Automake 1.15.1.  You should recreate
configure.ac:4: aclocal.m4 with aclocal and run automake again.
WARNING: 'automake-1.15' is probably too old.
         You should only need it if you modified 'Makefile.am' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'automake' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
```

**Comment by beckermr** [comment]
huzzah! thanks @isuruf!

File: recipe/build.sh [hunk 2]
```diff
@@ -1,9 +1,12 @@
-if [[ "`uname`" == "Linux" ]]
-then
-    export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+if [[ "$target_platform" == "amd64" || "$target_platform" == "win-64" ]]; then
```
Comment (isuruf) [inline recipe/build.sh:1]: This `amd64` hack is because of https://github.com/conda-forge/vc-feedstock/issues/16

Comment (beckermr) [reply recipe/build.sh:1]: See conda-forge/vc-feedstock#17

**Pull request merged**